app.js

```diff
@@ -1,68 +1,67 @@
-// var question = 'what\'s your name?';
-// var counter = 0;
-// var userName = prompt(question);
-// alert('Hello ' + userName + '! Welcome to my Page!');
-// console.log(userName);
+var question = 'what\'s your name?';
+var counter = 0;
+var userName = prompt(question);
+alert('Hello ' + userName + '! Welcome to my Page!');
+console.log(userName);
 
-// var guessFavMovie = prompt('Is my favorite movie Shawshank Redemption? Y/N');
-// guessFavMovie = guessFavMovie.toUpperCase();
-// if(guessFavMovie === 'Y' || guessFavMovie === 'YES') {
-//   alert('That is correct!');
-//   counter++;
-//   console.log(userName + ' answered : ' + guessFavMovie);
-// } else {
-//   alert('Incorrect. My favorite movie IS Shawshank Redemption');
-//   console.log(userName + ' answered ' + guessFavMovie);
-// }
+var guessFavMovie = prompt('Is my favorite movie Shawshank Redemption? Y/N').toUpperCase();
+if(guessFavMovie === 'Y' || guessFavMovie === 'YES') {
+  alert('That is correct!');
+  counter++;
+  console.log(userName + ' answered : ' + guessFavMovie);
+} else {
+  alert('Incorrect. My favorite movie IS Shawshank Redemption');
```

Comment:
All your else and else/ if blocks are indented wrong. You need one additional tab for each of those blocks.